feat: Prometheus Alertmanager Integration (#671)
target/goalert: web/src/app/documentation/IntegrationKeys.md

---

## Prometheus Alertmanager

Prometheus Alertmanager provides alerting functionality for checks as an IT automation.

To trigger an alert using Prometheus Alertmanager, follow these steps:

1. Within GoAlert, on the Services page, select the service you want to process the alert. Under Integration Keys:

   - Key Name: Enter a name for the key.
   - Key Type: Prometheus Alertmanager

>    - Click Add Key. Copy the generated URL and keep it handy, as you'll need it in a future step.

Reviewer: ```suggestion
   - Click Add Key. Copy the generated URL and keep it handy, as you'll need it for the next step.
```
